CHANGELOG.rst

```diff
@@ -69,6 +69,9 @@ Added
   to pants' use of PEX lockfiles. This is not a user-facing addition. #5713 #5724 #5726 #5725 #5732 #5733 #5737 #5738 #5758 #5751 #5774 #5776 #5777 #5782
   Contributed by @cognifloyd
 
+* Added publisher to ActionAlias to enable streaming ActionAlias create/update/delete events.
```

Comment:
This is in the changes made for stackstorm 3.8 - when should be in the new development section.